packages/DataGrid/stories/DataGrid.autofocus.stories.js

```diff
@@ -98,4 +98,4 @@ export function App() {
   );
 }
 
-storiesOf("DataGrid / regular", module).add("autofocus", () => <App />);
```

Comment:
DataGrid is one word because of the name of the component

Reply:
But the url for the showcase is /story/data-grid--showcase. I assume **-** between the word data and grid is equal to the space in the `storiesOf("Data Grid`

Reply:
ohhh ok, good point let it be like this, though now I wonder if we should set showcase as the main page or the Documentation page part, we can change that later easily. cc @oscarkwan

Reply:
I believe, we want Docs page to be the main page at the moment.

This is because all components will need a Docs page, but not all components require a showcase

Reply:
ok, we will do it later, right now we add showcase for all our components.